pkg/blockgen/profiles.go

```diff
@@ -26,6 +26,18 @@ func (p ProfileMap) Keys() (keys []string) {
 
 var (
 	Profiles = ProfileMap{
+		"realistic-key-k8s-1d-small": realisticK8s([]time.Duration{
```

Comment:
Hm, I think for our case it would be awesome to generate like... 30d (: with just 2-3 series. WDYT? (:

Reply:
We can create big blocks (e.g 2w) no problem

and name it `key-k8s-30s-tiny` with `}, 1*time.Hour, 1, 5),` (: